site/content/in-dev/unreleased/configuration.md

```diff
@@ -77,43 +77,46 @@ read-only mode, as Polaris only reads the configuration file once, at startup.
-| `polaris.metrics.tags.environment`                                                         | `prod`               | Define the environement tag in metrics.                                                                                                                                            |
-| `polaris.metrics.tags.region`                                                              | `us-west-2`          | Define the region tag in metrics.                                                                                                                                                  |
-| `polaris.tasks.max-concurrent-tasks`                                                       | `100`                | Define the max number of concurrent tasks.                                                                                                                                         |
-| `polaris.tasks.max-queued-tasks`                                                           | `1000`               | Define the max number of tasks in queue.                                                                                                                                           |
+| Configuration Property                                                                     | Default Value        | Description                                                                                                                                                                                           |
+|--------------------------------------------------------------------------------------------|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `polaris.persistence.type`                                                                 | `relational-jdbc`    | Define the persistence backend used by Polaris (`in-memory`, `relational-jdbc`, `eclipse-link`). See [Configuring Apache Polaris for Production)[{{% ref "configuring-polaris-for-production.md" %}}) | 
+| `polaris.persistence.relational.jdbc.max-retries`                                          | `1`                  | Total number of retries we expect the persistence to do before giving up                                                                                                                              |
+| `polaris.persistence.relational.jdbc.max_duaration_in_ms`                                  | `5000 ms`            | Time in ms since the first attempt the retries should be performed by jdbc.                                                                                                                           |
```

Comment:
suggestion: `Max time interval (ms) since the start of a transaction when retries can be attempted.`